src/examples/java/io/nats/examples/autobench/ThrottledBenchmark.java

```diff
@@ -73,10 +73,8 @@ public void execute(Options connectOptions) throws InterruptedException {
      * become slow consumers. The addition of this code was motivated by the asymetry in the java library between
      * publishers and subscribers, especially if UTF-8 subjects are enabled.
      * @param nc the connection
-     * @throws InterruptedException
      */
-	void adjustAndSleep(Connection nc) throws InterruptedException {
-
+	void adjustAndSleep(Connection nc) {
```

Comment:
Non client production code, only if already changing or fixing the class. BUT, this is an example, if you wanted to make a separate PR for just this, then I would go so far as to properly fix the exception handling too.